main.js

```diff
@@ -0,0 +1,81 @@
+const app = () => {
+  const song = document.querySelector(".song");
+  const play = document.querySelector(".play");
+  const outline = document.querySelector(".moving-outline circle");
+  const video = document.querySelector(".vid-container video");
+
+  // Sounds
+  const sounds = document.querySelectorAll(".sound-picker button");
+  // Time display
+  const timeDisplay = document.querySelector(".time-display");
+  const timeSelect = document.querySelectorAll(".time-select button");
+  // Get the length of outline
+  const outlineLength = outline.getTotalLength();
+  // Duration
+  let fakeDuration = 600;
```

Comment:
The name fakeDuration is a bit confusing, it is better to use someting like selectedDuration.